packages/headless/src/features/generated-answer/__snapshots__/generated-answer-analytics-actions.test.ts.snap

```diff
@@ -0,0 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logCopyGeneratedAnswer with the right payload 1`] = `
+[
+  "Qna.AnswerAction",
+  {
+    "action": "copyToClipboard",
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logDislikeGeneratedAnswer with the right payload 1`] = `
+[
+  "Qna.SubmitFeedback",
+  {
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+    "feedback": {
+      "liked": false,
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logGeneratedAnswerDetailedFeedback with the right payload 1`] = `
+[
+  "Qna.SubmitFeedback",
+  {
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+    "feedback": {
+      "details": "example details",
+      "liked": false,
+      "reason": "other",
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logGeneratedAnswerHideAnswers with the right payload 1`] = `
+[
+  "Qna.AnswerAction",
+  {
+    "action": "hide",
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logGeneratedAnswerShowAnswers with the right payload 1`] = `
+[
+  "Qna.AnswerAction",
+  {
+    "action": "show",
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logHoverCitation with the right payload 1`] = `
+[
+  "Qna.CitationHover",
+  {
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+    "citation": {
+      "id": "some-citation-id",
+    },
+    "citationHoverTimeInMs": 1234,
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logLikeGeneratedAnswer with the right payload 1`] = `
+[
+  "Qna.SubmitFeedback",
+  {
+    "answer": {
+      "id": "123",
+      "type": "CRGA",
+    },
+    "feedback": {
+      "liked": true,
+    },
+  },
+]
+`;
+
+exports[`generated answer analytics actions when analyticsMode is \`next\` should log #logOpenGeneratedAnswerSource with the right payload 1`] = `
+[
+  "Qna.CitationItemClick",
```

Comment:
I think it's `Qna.CitationClick`

Reply:
Indeed, I was just waiting for the latest version of `@coveo/relay-event-types` to implement this change without any ts issues 👍